pgx/src/array.rs

```diff
@@ -175,6 +182,21 @@ impl RawArray {
         self.data_offset() != 0
     }
 
+    /// May return null.
+    #[inline]
+    fn nulls_mut_ptr(&mut self) -> *mut u8 {
+        unsafe { pgx_ARR_NULLBITMAP(self.ptr.as_ptr()) }
+    }
```

Comment:
Does this need any `# safety` documentation? Or is it assumed here?

Reply:
It does.